ebuild-writing/common-mistakes/text.xml

```diff
@@ -78,26 +80,31 @@ someone reports a bug for your package, the <b>build log must always be verbose.
 <p>
 There are several ways to fix non-verbose build logs depending on the build system:
 </p>
-<p>
-For <c>cmake</c> based build systems it should be sufficient that the ebuild calls
-cmake-utils_src_compile which picks up the cmake-utils.eclass variable 'CMAKE_VERBOSE=1'
-by default. If you call emake directly for whatever reason, you can do 'emake VERBOSE=1'
-(note that cmake-utils_src_compile takes arguments as well which are passed to make).
-</p>
 
-<p>
-For <c>autotools</c> based build systems you can pass '--disable-silent-rules' to econf,
-or use EAPI 5 where that argument is passed automatically. 'emake V=1' should also work.
-</p>
+<ul>
+  <li>For <c>cmake</c> based build systems it should be sufficient that
```

Comment:
`<li>` (and `</li>`) goes in a line of its own (unless the whole `<li>...</li>` fits in one line).